libs/application/templates/reference-template/src/fields/CustomRepeater.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react'
+import { RepeaterProps } from '@island.is/application/core'
+import { Button, Typography } from '@island.is/island-ui/core'
+
+const CustomRepeater: FC<RepeaterProps> = ({ expandRepeater }) => {
+  return (
+    <>
+      <Typography as={'p'}>
```

Comment:
Can change this to Text component in future (since Typography will be deprecated soon).